### ISSUE TITLE
Integration with codeowner.md file

### ISSUE THREAD

**Issue by beaussan** [remorses/code-owners#2]
Hi,

Is it planed to generate a code owner file with github format with this tool?

Thanks

**Comment by remorses** [comment]
Yes, first I need to implement caching to make it faster between runs and think of a good way to get globs from a list of paths
